plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt

```diff
@@ -168,9 +180,16 @@ class GradleInspector(
                         addProgressListener(ProgressListener { logger.debug(it.displayName) })
                     }
 
-                    options[OPTION_JAVA_HOME]?.also {
+                    val javaHome = options[OPTION_JAVA_VERSION]?.let {
+                        JavaBootstrapper.installJdk("TEMURIN", it)
+                            .onFailure { e ->
+                                issues += createAndLogIssue(managerName, e.collectMessages())
+                            }.getOrNull()
```

Comment:
This should check if the currently running Java version matches the requested one and not install anything in this case.

Reply:
Good idea. Still, if a `javaHome` is also specified, it will be used in that case.